content/en/Platform Deep Dive/Pentests/Pentest Process/Methodologies/mobile.md

```diff
@@ -32,19 +32,106 @@ Learn more about these objectives from OWASP:
+files when you [Define Your Assets](/getting-started/assets/). You may also share this information in the private Slack channel
+for your pentest.
+
+### Methodology Details
 
 We follow an industry standard methodology primarily based on the OWASP Application Security
 Verification Standard (ASVS) and Testing Guide. Our team takes the following steps to ensure
 full coverage:
 
-- Reconnaissance
-  - Share the mobile application files
-    - Android: .apk
-    - iOS: .ipa
-- Automated and Manual Testing
-- Exploit Discovered Vulnerabilities
-- Report, triage, and retest
+Select the <i style="font-size:x-large; color: #0047AB" class="fas fa-chevron-right"></i> key for more information on each step.
+
+{{% expand "Target scope reconnaissance" %}}
+</br>
+Based on the Pentest Brief prepared by the client, Cobalt pentesters search for
+information about the targets and investigate the scope. This information
+includes:
+
+- Understanding workflows
+- Understanding business logic
+- Mapping the attack surface of the application
+
+Pentesters then confirm that they can:
+
+- Reach and scan the targets
+- Test the functionality of the application
+
+Our pentesters may use scanning tools such as:
+
+- MobSF
+- Frida
+- Apktool
+- Dex2Jar
+
+{{% /expand %}}
+
+{{% expand "Automated and Manual Testing" %}}
+</br>
+
+Our pentesters use a range of manual techniques and automated tools to ensure
+proper coverage. They analyze your mobile app dynamically. They also
+assess the archive as well as the local file.
+
+Our pentesters focus on:
+
+- Communication channels
+- Traffic that the application exchanges with external endpoints
+- Inter-Process Communication (IPC)
+
+Our pentesters may also reverse-engineer the application for insight, and to
+try to access sensitive data.
+
+For backend tests, they use [API Pentest Methodologies](platform-deep-dive/pentests/pentest-process/methodologies/api-methodologies/).
+
+Our pentesters may use scanning tools such as:
+
+- MobSF
+- Frida
+- Apktool
+- Dex2Jar
+
+{{% /expand %}}
+
+{{% expand "Exploit Discovered Vulnerabilities" %}}
+</br>
+
+When our pentesters discover a vulnerability, they use various techniques to measure the impact on
+the following aspects of your  ata:
+
+- Confidentiality
+- Integrity
+- Availability
+
+Our pentesters use various privilege escalation methods to impersonate different users, possibly
+to impact the security posture of the application.
+
+Our pentesters may use scanning tools such as:
+
+- MobSF
+- Frida
+- Apktool
+- Dex2Jar
+
+{{% /expand %}}
+
+{{% expand "Report, Triage, and Restest" %}}
+</br>
+
+Pentesters report and triage all findings as they work. We share details through our online platform. 
+We encourage you to address critical findings as we discover them. Our pentesters can retest, based
+on your [PtaaS Tier](/platform-deep-dive/credits/ptaas-tiers/).
```

Comment:
```suggestion
Pentesters report and triage all findings as they work. We share details through our online platform. 
We encourage you to address critical findings as we discover them. Our pentesters can retest findings, based
on your [PtaaS tier](/platform-deep-dive/credits/ptaas-tiers/).
```

> We share details through our online platform.

Should we specify where exactly customers can view findings?

> based on your [PtaaS Tier](/platform-deep-dive/credits/ptaas-tiers/)

I'm not sure we need to add a link to the new article about PtaaS tiers. I think that in most cases we should still redirect users to https://www.cobalt.io/pentest-pricing. What do you think?